client/src/pages/addFlight/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './addflight.css';
+import '../../App.css';
+import Durationcard from '../../components/durationcard';
+import Select from '../../components/select';
+import Input from '../../components/input';
+import Timebutton from '../../components/timebutton';
+import Button from '../../components/button';
+import Header from '../../components/header';
+import Sidenav from '../../components/sidenav';
+
+class UpdateFlight extends Component {
+  render() {
+    return (
+        <div className="addflight-container">
+          <Header />
+          <div className="container-content">
+            <div className="sub-container-sidenav-form">
+              <Sidenav />
+              <div className="sub-container-form">
+                <Select labelText='Origin' firstItem='from' />
+                <Select labelText='Destination' firstItem='to' />
+                <Input labelText='Flight date' placeholder='flight date' />
```

Comment:
On input change? What happens?

Reply:
we are just drawing the page with JSX and CSS
not diving in react yet and get the values to compile